tests/testthat/test-get_accessibility_2000.R

```diff
@@ -0,0 +1,16 @@
+test_that(".get_accessibility_2000 works", {
+  skip_on_cran()
```

Comment:
I think that this does not need to be skipped on CRAN since the code is not relying on a connection to the remote server, correct?

Reply:
Since you are constructing the tile by hand, `get_traveltime_2000()` does not rely on a connection to the server. It is thus ok to test this function on CRAN.